Source/swiftlint/Commands/ShowDocsCommand.swift

```diff
@@ -0,0 +1,29 @@
+import Commandant
+import Foundation
+
+struct ShowDocsCommand: CommandProtocol {
+    let verb = "show-docs"
+    let function = "Open `SwiftLint Framework Docs` on web browser"
+
+    func run(_ options: NoOptions<CommandantError<()>>) -> Result<(), CommandantError<()>> {
+        let url = URL(string: "https://realm.github.io/SwiftLint")!
+        open(url)
+        return .success(())
+    }
+}
+
+private extension ShowDocsCommand {
+    func open(_ url: URL) {
+        let process = Process()
+        process.launchPath = "/usr/bin/env"
+        let command: String = {
+            #if os(Linux)
+            return "xdg-open"
```

Comment:
I don't think this works on every Linux distribution 🤔

Reply:
I choose the most universal command as far as I know.
I'm not expert with Linux, and don't know if there is more better solution😔